src/__tests__/surveys.test.ts

```diff
@@ -147,7 +162,12 @@ describe('surveys', () => {
             },
         } as unknown as PostHog
```

Comment:
to keep this PR smaller, i'm not replacing the mock posthog here
(we really should be, but it can wait)